profiles/fscloud/variables.tf

```diff
@@ -20,7 +25,7 @@ variable "pg_version" {
 }
 
 variable "region" {
-  description = "The region postgresql is to be created on. The region must support BYOK if key_protect_key_crn is used"
+  description = "The region postgresql is to be created on. The region must support KYOK if kms_key_crn is used"
```

Comment:
`kms_key_crn` is a required variable for FSCloud profile, so remove the text `if kms_key_crn is used`